src/commands/force/source/delete.ts

```diff
@@ -221,6 +322,14 @@ export class Delete extends DeployCommand {
         }
       });
 
+      if (this.mixedDeployDelete.delete.length) {
```

Comment:
if we're going go overwrite `local` with the `mixedDeployDelete` value, do we need to do line 318?